src/effector/createUnit.js

```diff
@@ -279,7 +278,7 @@ export function createStore<State>(
       } else {
         eventOrFn(store.getState())
       }
-      return watchUnit(store, eventOrFn)
+      return watchUnit(store)(eventOrFn)
```

Comment:
No `fn()()`, please

Reply:
What about like this? :)
```suggestion
      const watchStore = watchUnit(store)
      return watchStore(eventOrFn)
```

Reply:
Nice try 😄But no, no `fn()()`, please

Reply:
Hey, you have `fn()()` inside `createEffect` ;) And that wasn't my work :)

![image](https://user-images.githubusercontent.com/6583994/78917823-1ab66480-7a98-11ea-99dd-8f5a7af03433.png)